orderer/common/multichannel/registrar_test.go

```diff
@@ -254,6 +255,38 @@ func TestNewRegistrar(t *testing.T) {
 	})
 }
 
+func TestNewRegistrarWithFileRepo(t *testing.T) {
```

Comment:
Not related to this PR alone. In general, this package has no coverage for errors or panics. Please consider covering that if possible in the future PRs.